src/Serilog.Sinks.ElmahIo/LoggerConfigurationElmahIOExtensions.cs

```diff
@@ -55,18 +56,39 @@ public static class LoggerConfigurationElmahIoExtensions
         /// <param name="formatProvider">Supplies culture-specific formatting information, or null.</param>
         /// <returns>Logger configuration, allowing configuration to continue.</returns>
         /// <exception cref="ArgumentNullException">A required parameter is null.</exception>
+        [Obsolete("Use the overload accepting ElmahIoSinksOptions")]
         public static LoggerConfiguration ElmahIo(
             this LoggerSinkConfiguration loggerConfiguration,
             string apiKey,
             Guid logId,
             LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum,
             IFormatProvider formatProvider = null)
+        {
+            return ElmahIo(loggerConfiguration, new ElmahIoSinkOptions
+            {
+                ApiKey = apiKey,
+                LogId = logId,
+                MinimumLogEventLevel = restrictedToMinimumLevel,
+                FormatProvider = formatProvider,
+            });
+        }
+
+        /// <summary>
+        /// Adds a sink that writes log events to elmah.io. 
+        /// </summary>
+        /// <param name="loggerConfiguration">The logger configuration.</param>
+        /// <param name="options"></param>
+        /// <returns>Logger configuration, allowing configuration to continue.</returns>
+        /// <exception cref="ArgumentNullException">A required parameter is null.</exception>
+        public static LoggerConfiguration ElmahIo(
+            this LoggerSinkConfiguration loggerConfiguration,
+            ElmahIoSinkOptions options)
         {
             if (loggerConfiguration == null) throw new ArgumentNullException(nameof(loggerConfiguration));
             return loggerConfiguration.Sink(
-                new ElmahIoSink(formatProvider, apiKey, logId),
-                restrictedToMinimumLevel);
+                new ElmahIoSink(options),
+                restrictedToMinimumLevel: options.MinimumLogEventLevel ?? LevelAlias.Minimum,
```

Comment:
Specify this default in the xml comments?

Reply:
👍